### ISSUE TITLE
[BUG] toVFile writeSync produces no different contents 

### ISSUE THREAD

**Issue by dreamer2q** [vfile/to-vfile#15]
---
name: 🐛 Bug report
about:  toVFile writeSync produces no different contents
---

### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 

Affected packages and versions: 7.0.0

### Steps to reproduce

![image](https://user-images.githubusercontent.com/56211747/117825449-ec5e1680-b2a1-11eb-877b-e6c52b791017.png)

### Expected behavior

![image](https://user-images.githubusercontent.com/56211747/117826535-cf761300-b2a2-11eb-83b7-10a2970a190e.png)

### Actual behavior

By using to-vfile to read and write back to a file with modified contents !

In this picture, you can see, toVFile is used to read file (`assets/test2.html`) and write back to (`dist/a.html`) with contents modified.

And in the debug console, you can see the first line of modified contents.

BUT IN THE FILE (`dist/a.html`) which is created by toVFile.writeSync(file),
YOU WILL FIND OUT THAT THIS IS JUST THE SAME AS FILE (`assets/test2.html`)

IN SHORT, toVFile works nothing at all but create the same file as it previously read.

![image](https://user-images.githubusercontent.com/56211747/117826613-e0bf1f80-b2a2-11eb-8e87-3b3db7a704df.png)

**Comment by dreamer2q** [comment]
![image](https://user-images.githubusercontent.com/56211747/117827861-ebc67f80-b2a3-11eb-82ef-365fa67ae7d5.png)


I found out a solution which perfectly solved this problems, cheers !

**Comment by wooorm** [comment]
Please fix your issue to match our template
